ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/Wizard/ScanConfigWizardForm.tsx

```diff
@@ -30,12 +32,25 @@ function ScanConfigPage(): ReactElement {
     const { clusters, isLoading: isFetchingClusters } = useFetchClustersForPermissions([
         'Compliance',
     ]);
+    const [isCreating, setIsCreating] = useState(false);
+    const [createScanConfigError, setCreateScanConfigError] = useState('');
 
     const listQuery = useCallback(() => listComplianceProfiles(), []);
     const { data: profiles, loading: isFetchingProfiles } = useRestQuery(listQuery);
 
-    function onCreate() {
-        // TODO: create scan
+    async function onCreate() {
+        setIsCreating(true);
+        setCreateScanConfigError('');
+        const complianceScanConfig = convertFormikToScanConfig(formik.values);
+
+        try {
+            await createScanConfig(complianceScanConfig);
+            history.push(complianceEnhancedScanConfigsBasePath);
```

Comment:
An observation, not a change request: depending on how many ways to reach a form or wizard page, an alternative that might work for both Save and Cancel is `history.goBack()` which has side effect to simplify the history that you see when you right-click browser Back button.